src/GenerateNotice.Nuget/build/Azure.Deployments.Internal.GenerateNotice.targets

```diff
@@ -3,19 +3,41 @@
   <PropertyGroup>
     <GenerateNoticeRetryCount Condition="'$(GenerateNoticeRetryCount)' == ''">3</GenerateNoticeRetryCount>
     <GenerateNoticeBatchSize Condition="'$(GenerateNoticeBatchSize)' == ''">100</GenerateNoticeBatchSize>
+    
```

Comment:
nit: Extra new line? Or intentional?

Reply:
Yeah, did that to visually group the properties together. The bottom ones are related to the "local file" functionality.